dockernet/config/ica_host.json

```diff
@@ -17,6 +17,7 @@
                     "/cosmos.staking.v1beta1.MsgCancelUnbondingDelegation",
                     "/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward",
                     "/cosmos.distribution.v1beta1.MsgSetWithdrawAddress",
+                    "/cosmos.distribution.v1beta1.MsgFundCommunityPool",
```

Comment:
This is the whitelist of which msg types are allowed?  Specifically which msg types the host will listen for and in this case the host is DYDX and stride is the client